packages/base/src/panelview/leftpanel.tsx

```diff
@@ -34,16 +33,16 @@ export class LeftPanelWidget extends SidePanel {
     this.addWidget(properties);
 
     this._handleFileChange = () => {
-      header.title.label = this._currentContext?.localPath || '-';
+      header.title.label = this._currentContext?.filePath || '-';
```

Comment:
I'd rename this property, since it's the model now

```suggestion
      header.title.label = this._currentModel?.filePath || '-';
```